ui/app/components/ui/sender-to-recipient/sender-to-recipient.component.js

```diff
@@ -19,6 +19,7 @@ export default class SenderToRecipient extends PureComponent {
     senderAddress: PropTypes.string,
     recipientName: PropTypes.string,
     recipientAddress: PropTypes.string,
+    recipientNickname: PropTypes.object,
```

Comment:
I think this might need to be `PropTypes.string`:

```suggestion
    recipientNickname: PropTypes.string,
```

Reply:
It most definitely should be a `PropTypes.string`